Base/Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN echo "${TZ}" > /etc/timezone \
 RUN useradd seluser \
          --shell /bin/bash  \
          --create-home \
-  && usermod -a -G sudo seluser \
+  && usermod -a -G sudo,audio,video seluser \
```

Comment:
Just out of curiosity, why are these permissions needed?

Reply:
I assumed that **pulseaudio** needs audio permissions, but as I read in the documentation, it seems that it is not necessary.

The `video` group was added while trying to support hardware acceleration for browsers and is currently probably useless, so I forgot to remove it.

I am going to check if they are really necessary and I will come back again.

Reply:
I see you removed it, I assume it was not needed, thanks!